### PR TITLE
Veyis practice app organized

### PR DESCRIPTION
**Pull request by veyisTurgut**
* I completed my testcases. You can check them out before writing your own tests. 
* I wrote testcases for both endpoints and helper methods. 
* There is a problem about importing. Run in Pycharm with marking directory as "Sources Root" to avoid this problem.

Finally changed the name of the practice-app.py to main.py since python cant import files that contain "-".

### CODE REVIEW

File: practice-app/src/main/practice-app.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def get_tasks():
 
 @app.route('/books/', methods=['GET'])
 def get_books():
+    print(request.args)
```
Comment (veyisTurgut) [inline practice-app/src/main/practice-app.py:32]: Ah, I forgot to delete this print. Sorry about that.

**Comment by veyisTurgut** [comment]
Frontend is also completed. I guess my job is done here.

**Comment by gokayyildiz** [comment]
`def add_book_from_user(book):
    # connect to Database
    try:
        con = sqlite3.connect("./sqlfiles/practice-app.db")
        cur = con.cursor()`


Why don't you use the full path you defined above in there

**Comment by veyisTurgut** [comment]
> `def add_book_from_user(book): # connect to Database try: con = sqlite3.connect("./sqlfiles/practice-app.db") cur = con.cursor()`
> 
> Why don't you use the full path you defined above in there

You are right, I guess it worked this way :D.

**Comment by veyisTurgut** [comment]
There is a minor bug, it's mentioned here: https://github.com/bounswe/2021SpringGroup12/issues/155 . I will fix it and open the request again

**Pull request closed without merging**